.github/workflows/dev-image-update.yaml

```diff
@@ -20,22 +20,31 @@ jobs:
           repository: 'broadinstitute/datarepo-helm-definitions'
           token: ${{ secrets.HELM_REPO_TOKEN }}
           path: datarepo-helm-definitions
+      - name: "Bump the tag to a new version"
+        uses: broadinstitute/datarepo-actions@0.8.1
+        with:
+          actions_subcommand: 'bumper'
+          role_id: ${{ secrets.ROLE_ID }}
+          secret_id: ${{ secrets.SECRET_ID }}
+          version_file_path: datarepo-client/build.gradle
+          version_variable_name: version
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: "Build new delevop docker image"
-        uses: broadinstitute/datarepo-actions@0.6.0
+        uses: broadinstitute/datarepo-actions@0.8.1
         with:
           actions_subcommand: 'gradlebuild'
           role_id: ${{ secrets.ROLE_ID }}
           secret_id: ${{ secrets.SECRET_ID }}
       - name: "Check and edit Helm definition for dev"
-        uses: broadinstitute/datarepo-actions@0.6.0
+        uses: broadinstitute/datarepo-actions@0.8.1
         with:
           actions_subcommand: 'deploytagupdate'
           helm_env_prefix: dev
       - name: "Check and edit Helm definition for integration"
-        uses: broadinstitute/datarepo-actions@0.6.0
+        uses: broadinstitute/datarepo-actions@0.8.1
         with:
           actions_subcommand: 'deploytagupdate'
-          helm_env_prefix: 'integration-1,integration-2,integration-3,integration-4,integration-5'
+          helm_env_prefix: 'integration-4,integration-5'
```

Comment:
Why is this changing to only impact int-4 & 5?

Reply:
We shouldn't be auto-deploying to int-1, int-2, or int-3 when we merge a branch. It means that any integration tests that might be running against those envs will not be testing the right code any more.

Reply:
We reserve int-4 and int-5 for UI integration tests, and so we want those to always have the latest dev image.